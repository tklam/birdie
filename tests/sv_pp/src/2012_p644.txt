`define msg(x,y) `"x: `\`"y`\`"`"
`msg(left side,right side)

`define append(f) f``_master
`append(clock)
