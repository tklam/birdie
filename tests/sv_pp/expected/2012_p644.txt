
"left side: \"right side\""


clock_master
